pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -332,7 +332,9 @@ def _set_dimensionless_parameters(self):
         )
 
         # SEI parameters
-        self.alpha_sei = pybamm.Parameter("Inner SEI reaction proportion")  # was 0.5
+        self.inner_prop = pybamm.Parameter("Inner SEI reaction proportion")  # was 0.5
```

Comment:
would be good to have `_sei` in this parameter name

Reply:
Changed to inner_sei_proportion